lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -136,6 +136,9 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
                {ConvolutionNode::BiasIdx}) &&
            (NI.getInElemTy(ConvolutionNode::BiasIdx) == ElemKind::Int32QTy);
 
+  case Kinded::Kind::ChannelwiseQuantizedConvolutionNodeKind:
+    return true;
```

Comment:
I think you should expand on this, no? For example it's possible we could create a `ChannelwiseQuantizedConvolutionNode` and then the FP16 converter could modify some input to FP16. See RWQ-SLWS -- it's a very specialized Node but we still check specific element kinds on the inputs/outputs.